Content.Server/_DV/CosmicCult/Abilities/CosmicBlankSystem.cs

```diff
@@ -0,0 +1,131 @@
+using Content.Server.Popups;
+using Content.Shared._DV.CosmicCult;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Components.Examine;
+using Content.Shared.DoAfter;
+using Content.Shared.Effects;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Mind;
+using Content.Shared.Mind.Components;
+using Content.Shared.NPC;
+using Content.Shared.Stunnable;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Player;
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+
+namespace Content.Server._DV.CosmicCult.Abilities;
+
+public sealed class CosmicBlankSystem : EntitySystem
+{
+    [Dependency] private readonly CosmicCultSystem _cult = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly PopupSystem _popup = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly SharedColorFlashEffectSystem _color = default!;
+    [Dependency] private readonly SharedDoAfterSystem _doAfter = default!;
+    [Dependency] private readonly SharedMindSystem _mind = default!;
+    [Dependency] private readonly SharedStunSystem _stun = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+    }
+
+        private void OnCosmicBlank(Entity<CosmicCultComponent> uid, ref EventCosmicBlank args)
+    {
+        if (HasComp<CosmicCultComponent>(args.Target) || HasComp<CosmicMarkBlankComponent>(args.Target) || HasComp<BibleUserComponent>(args.Target) || HasComp<ActiveNPCComponent>(args.Target))
+        {
+            _popup.PopupEntity(Loc.GetString("cosmicability-generic-fail"), uid, uid);
+            return;
+        }
+        if (args.Handled)
+            return;
+
+        var doargs = new DoAfterArgs(EntityManager, uid, uid.Comp.CosmicBlankDelay, new EventCosmicBlankDoAfter(), uid, args.Target)
+        {
+            DistanceThreshold = 1.5f,
+            Hidden = false,
+            BreakOnDamage = true,
+            BreakOnMove = true,
+            BreakOnDropItem = true,
+        };
+        args.Handled = true;
+        _doAfter.TryStartDoAfter(doargs);
+        _popup.PopupEntity(Loc.GetString("cosmicability-blank-begin", ("target", Identity.Entity(uid, EntityManager))), uid, args.Target);
+    }
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        var shuntQuery = EntityQueryEnumerator<InVoidComponent>();
+        while (shuntQuery.MoveNext(out var uid, out var comp))
+        {
+            if (_timing.CurTime >= comp.ExitVoidTime)
+            {
+                if (!TryComp<MindContainerComponent>(uid, out var mindContainer))
+                    continue;
+                var mindEnt = mindContainer.Mind!.Value;
+                var mind = Comp<MindComponent>(mindEnt);
```

Comment:
null supression would crash if someone is ssd, use `if (!_mind.TryGetMind(uid, out var mindEnt, out var mind))` instead